server/src/test/java/io/spine/server/event/EventStoreShould.java

```diff
@@ -158,14 +155,25 @@ public void read_events_by_type() {
         final AtomicBoolean done = new AtomicBoolean(false);
         final Collection<Event> resultEvents = newConcurrentHashSet();
         eventStore.read(query, new ResponseObserver(resultEvents, done));
-        if (!done.get()) {
-            fail("Please use the MoreExecutors.directExecutor in EventStore for tests.");
-        }
+        assertDone(done);
 
         assertSize(2, resultEvents);
         assertContainsAll(resultEvents, taskAdded1, teasAdded2);
     }
 
+    /**
+     * Checks that the event storage is exposed to Beam-based catch-up code which is in the same
```

Comment:
These tests are in the `server` module to preserve the API. Presumably, `server-deployment` is going to live in a separate repository.  Once `server-deployment` is moved, there would be no usages in the `server` module in the scenario you suggest. It would be easy to just drop the code.